scripts/ingest_regress_test_result.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+
+import psycopg2
+
+CREATE_TABLE = """
+CREATE TABLE IF NOT EXISTS regress_test_results (
+    id SERIAL PRIMARY KEY,
+    reference CHAR(255),
+    revision CHAR(40),
+    build_type CHAR(16),
+    data JSONB
+)
+"""
+
+
+def err(msg):
+    print(f"error: {msg}")
+    sys.exit(1)
+
+
+@contextmanager
+def get_connection_cursor():
```

Comment:
Probably a good idea to move utility functions to separate module. It can be injected via adjusting PYTHONPATH. Or we can move this to separate package and runsscripts via poetry run. Not a big problem right now, but at some point we’ll need something like that

Reply:
Agree. Will revise once we have another script for uploading data or if we need to make major changes to these two.